src/material/chips/testing/chip-harness.ts

```diff
@@ -31,7 +31,7 @@ export class MatChipHarness extends ComponentHarness {
 
   /** Gets the text of the chip. */
   async getText(): Promise<string> {
-    return (await this.host()).text();
+    return (await this.host()).text({exclude: '.mat-chip-avatar, .mat-chip-trailing-icon'});
```

Comment:
Should also exclude `'.mat-icon'`

Reply:
@crisbeto do you know if we expect people to use mat-icon directly in mat-chip without marking it as an avatar or trailing icon?

Reply:
I believe that all our examples go through the avatar or the trailing icon directives. I've added `.mat-icon` to the selector anyway since it doesn't require any maintenance on our end.